### PR TITLE
[W5.11][W09-A4]Charmaine Lee

### PR DESCRIPTION
**Pull request by charmaineleehc**
New feature: find command is now case-insensitive

### CODE REVIEW

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -47,14 +47,31 @@ public CommandResult execute() {
...
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            Set<String> lowerCaseWordsInName = convertSetOfStringsToLowerCase(wordsInName);
+            if (!Collections.disjoint(lowerCaseWordsInName, lowerCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
```
Comment (shawnclq) [inline src/seedu/addressbook/commands/FindCommand.java:62]: Good comment!

File: test/expected.txt [hunk 2]
```diff
@@ -213,8 +213,9 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo Tags: [secretive]
```
Comment (shawnclq) [inline test/expected.txt:216]: Good job updating test cases!

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -47,14 +47,31 @@ public CommandResult execute() {
...
+     * Convert all strings in a set to lower case.
+     *
+     * @param setOfStrings
+     * @return set of strings in lower case
+     */
+    private Set<String> convertSetOfStringsToLowerCase(Set<String> setOfStrings) {
```
Comment (goweiwen) [inline src/seedu/addressbook/commands/FindCommand.java:68]: Can this method be static?

File: src/seedu/addressbook/commands/FindCommand.java [hunk 4]
```diff
@@ -47,14 +47,31 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        Set<String> lowerCaseKeywords = convertSetOfStringsToLowerCase(keywords);
```
Comment (goweiwen) [inline src/seedu/addressbook/commands/FindCommand.java:50]: Can this variable be final? Keeping variables final can help prevent the developer from accidentally reassigning it.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 5]
```diff
@@ -47,14 +47,31 @@ public CommandResult execute() {
...
+            Set<String> lowerCaseWordsInName = convertSetOfStringsToLowerCase(wordsInName);
+            if (!Collections.disjoint(lowerCaseWordsInName, lowerCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Convert all strings in a set to lower case.
```
Comment (goweiwen) [inline src/seedu/addressbook/commands/FindCommand.java:63]: Is this actually converting the strings in a set? This implies `setOfStrings` is being mutated, but this method is creating a new set and returning it.

**Review by goweiwen — COMMENTED** [review]
Good job completing this LO!

I have left some comments. Please close this PR after reading them.

**Pull request closed without merging**